### PR TITLE
Revert adding new collapsible-menu-view prop for 'More Button' to switch variants, prevent MVB

### PR DESCRIPTION
**Pull request by neilpfeiffer**
### Summary
Reverts portions of #943 after change in directions:
- removes need for new `menuButtonVariant` prop (no longer needed), 
**_>>> if merged before next npm release it will prevent a Major Version Bump <<<_** 
- reverts order for props to [original](https://github.com/cerner/terra-framework/commit/6cf7f8722f57836915c06589297b30efd09fb1df): (`children`, `menuWidth`, `boundingRef`),
- updates 'More' menu button style to be `utility` for all uses,
- changes the icon to match OCS `rollup` icon,
- adds styling adjustments for [icon alignment in IE (#2523)](https://github.com/cerner/terra-core/pull/2523/files#diff-b2ae2172c829371a523555d2e0626b06) to match action-header

### Additional Details
replaces PR #965

- For reference - [comparison of changes relative to master(commit:1613dd8)](https://github.com/cerner/terra-framework/compare/1613dd8dcdbdd7c85a5ebda9e4d463b2e7ad9d48...revert-adding-new-collapsible-menu-prop) from before 943(commit:6cf7f87)

### CODE REVIEW

**Comment by neilpfeiffer** [comment]
Ready for review

File: packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss [hunk 1]
```diff
@@ -15,19 +15,26 @@
...
+      background: var(--terra-collapsible-menu-view-menu-button-icon-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#1c1f21" d="M18.6 24c0-3 2.4-5.4 5.4-5.4s5.4 2.4 5.4 5.4-2.4 5.4-5.4 5.4c-3 0-5.4-2.4-5.4-5.4zm0-18.6C18.6 2.4 21 0 24 0c3 0 5.4 2.4 5.4 5.4 0 3-2.4 5.4-5.4 5.4-3 0-5.4-2.4-5.4-5.4zm0 37.2c0-3 2.4-5.4 5.4-5.4s5.4 2.4 5.4 5.4c0 3-2.4 5.4-5.4 5.4-3 0-5.4-2.4-5.4-5.4z"/></svg>'));
+      background-position: center;
       background-repeat: no-repeat;
-      background-size: cover;
+      background-size: auto;
       display: inline-block;
+      position: relative;
+      top: var(--terra-collapsible-menu-view-menu-button-icon-top, 0);
+      vertical-align: var(--terra-collapsible-menu-view-menu-button-icon-vertical-align, -0.14285rem);
+      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```
Comment (emilyrohrbough) [inline packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss]: Should we be adding vender prefixes to the styles?

Comment (neilpfeiffer) [reply packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss]: That is needed for IE specific SVG/icon and text alignment details we spent many hours on here:  [#2523](https://github.com/cerner/terra-core/pull/2523/files#diff-b2ae2172c829371a523555d2e0626b06)

Comment (emilyrohrbough) [reply packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss]: Can we add some inline comments regarding this so future engineers/unfamiliar people know why we've added these?

Comment (neilpfeiffer) [reply packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss]: Added comment https://github.com/cerner/terra-framework/pull/966/commits/739d850974e32564a29991f569a06ae34445548e

**Pull request merged**